app/templates/components/generic-chart.hbs

```diff
@@ -0,0 +1,9 @@
+<div class='chart'></div>
```

Comment:
I think it would be better to have these checkboxes in the settings panel. Better yet have it defined by the query.